src/csproj.js

```diff
@@ -95,7 +115,7 @@ const parseProject = (filePath, options) => {
             } else {
               return parsePackages(packagesLocation)
                 .then(packages => {
-                  result.packages = packages || [];
+                  result.packages = mergePackages(result.packages, packages);
```

Comment:
While playing around with msbuild a bit last week, I noticed that it would only try to restore from packages.config if the csproj file didn't have any `PackageReference`s in it. Based on that, I think we should take the same approach here. Can we revert this merge operation, and instead expand the check above to only read from packages.config if `deepParse` is requested and `result.packages` doesn't have anything in it?